CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 IMPROVEMENTS:
 * Add support for externally managed Group Member IDs to Vault Identity Group: ([#1630](https://github.com/hashicorp/terraform-provider-vault/pull/1630))
 * Support configuring vault version handling: ([#1646](https://github.com/hashicorp/terraform-provider-vault/pull/1646))
+* Add Redis database secrets engine support: ([#1659](https://github.com/hashicorp/terraform-provider-vault/pull/1659))
```

Comment:
No need to update the changelog yet. We will generate it from the PR description when we cutting the 3.11.0 release.